edge-hub/watchdog/linux/src/main.rs

```diff
@@ -0,0 +1,114 @@
+use nix::sys::signal::{self, Signal};
```

Comment:
seems like it is still not sorted

Reply:
Does cargo fmt not sort? I need to check